runtime/compiler/aarch64/codegen/J9MemoryReference.hpp

```diff
@@ -45,15 +46,26 @@ namespace ARM64
 
 class OMR_EXTENSIBLE MemoryReference : public OMR::MemoryReferenceConnector
    {
+   flags8_t _j9Flags;
```

Comment:
Where is `_j9Flags` initialized (to 0?)

Reply:
I think the default constructor of `flags8_t` will run, but added `_j9Flags` to the member initialization list in J9MemoryReference's constructors.

Reply:
Ah, I didn't notice `flags_t` was a class in OMR.
Thank you for the change anyway.